### ISSUE TITLE
he loss of my training and testing has been declining. I tested it with epoch 85th, but the effect is not so good. How many times does this epoch have to be better?

### ISSUE THREAD

**Issue by whaozl** [dongjun-Lee/text-summarization-tensorflow#15]

**Comment by dongjun-Lee** [comment]
Do you mean 85 batches? To train 85 epochs, it will take too much time.
I trained two epochs to generate sample output as [here](https://github.com/dongjun-Lee/text-summarization-tensorflow#sample-summary-output).
For example,
```
python train.py --num_epochs=2
```

**Comment by pnagula** [comment]
please let me know if you have run training for 2 epochs on all the gigaword data 3.8M rows or just 50K sample rows
